source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/update/PropertyContactEditForm.tsx

```diff
@@ -0,0 +1,142 @@
+import { Formik, FormikProps } from 'formik';
+import React, { useMemo, useState } from 'react';
+import styled from 'styled-components';
+
+import { Input, Select, SelectOption, TextArea } from '@/components/common/form';
+import { ContactInputContainer } from '@/components/common/form/ContactInput/ContactInputContainer';
+import ContactInputView from '@/components/common/form/ContactInput/ContactInputView';
+import LoadingBackdrop from '@/components/common/LoadingBackdrop';
+import { Section } from '@/components/common/Section/Section';
+import { SectionField } from '@/components/common/Section/SectionField';
+import { StyledSummarySection } from '@/components/common/Section/SectionStyles';
+import { useOrganizationRepository } from '@/features/contacts/repositories/useOrganizationRepository';
+import { IContactSearchResult } from '@/interfaces';
+import { Api_OrganizationPerson } from '@/models/api/Organization';
+import { Api_PropertyContact } from '@/models/api/Property';
+import { formatApiPersonNames, formatContactSearchResult } from '@/utils/personUtils';
+
+import { PropertyContactFormModel } from './models';
+import { PropertyContactEditFormYupSchema } from './validation';
+
+export interface IPropertyContactEditFormProps {
+  isLoading: boolean;
+  propertyContact: Api_PropertyContact;
+  onSave: (apiModel: Api_PropertyContact) => void;
+}
+
+export const PropertyContactEditForm = React.forwardRef<
+  FormikProps<any>,
+  IPropertyContactEditFormProps
```

Comment:
I think this needs a yup schema otherwise:
1) the purpose is supposed to be required but that isn't being enforced
2) the purpose has a character limit that should also be enforced.